lib/diagnostic.rb

```diff
@@ -102,6 +119,10 @@ def eat_meat(food)
       puts 'Yuck!'
     end
   end
+
+class Lion < Cat
+  include Carnivorous
+
 end
 
```

Comment:
In direct inheritance, you're limited to inheriting from a single source.
In composition, you can draw from as many modules as you like.